src/apps/saddle/saddle.module.ts

```diff
@@ -16,18 +17,17 @@ import { SaddleAppDefinition, SADDLE_DEFINITION } from './saddle.definition';
   appId: SADDLE_DEFINITION.id,
   imports: [SynthetixAppModule],
   providers: [
-    SaddleAppDefinition,
-    SaddleContractFactory,
-    // Helpers
     CurvePoolTokenHelper,
     CurveVirtualPriceStrategy,
-    SaddleOnChainCoinStrategy,
-    SaddleOnChainReserveStrategy,
-    // Ethereum
     EthereumSaddleBalanceFetcher,
-    EthereumSaddlePoolTokenFetcher,
     EthereumSaddleCommunalFarmContractPositionFetcher,
     EthereumSaddleMiniChefV2FarmContractPositionFetcher,
+    EthereumSaddlePoolTokenFetcher,
+    EvmosSaddlePoolTokenFetcher,
```

Comment:
You'll need to add `EvmosSaddleCommunalFarmContractPositionFetcher` and `EvmosSaddleMiniChefV2FarmContractPositionFetcher` as well.